packages/components/src/components/AddFileButton.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <v-button class="clear add-file__button" size="small" kind="ghost">
+    <PlusIcon />Add content...
+  </v-button>
+</template>
+
+<script lang="ts">
+//@ts-nocheck
+import PlusIcon from '../assets/plus.svg';
+export default {
+  name: 'v-chat-search',
+  components: {
+    PlusIcon,
```

Comment:
`v-button` is not registered so the button does not render

Reply:
![image](https://github.com/user-attachments/assets/2c6e2313-0c40-445a-a1f4-84be48214958)